app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/analysisForm/formField/formField.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FieldProvider } from 'components/fields/fieldProvider';
+import { FieldErrorHint } from 'components/fields/fieldErrorHint';
+import styles from './formField.scss';
+
+const cx = classNames.bind(styles);
+
+export class FormField extends PureComponent {
+  static propTypes = {
+    containerClasses: PropTypes.string,
+    labelClasses: PropTypes.string,
+    inputWrapperClasses: PropTypes.string,
+    descriptionClasses: PropTypes.string,
+    withErrorHint: PropTypes.bool,
+    withPercent: PropTypes.bool,
+    isRadio: PropTypes.bool,
+    label: PropTypes.string,
+    descriptions: PropTypes.array,
+    onChange: PropTypes.func,
+    formatValue: PropTypes.func,
+    parseValue: PropTypes.func,
+    fieldName: PropTypes.string,
+    children: PropTypes.any,
+  };
+
+  static defaultProps = {
+    containerClasses: '',
+    labelClasses: '',
+    inputWrapperClasses: '',
+    descriptionClasses: '',
+    withErrorHint: false,
+    withPercent: false,
+    isRadio: false,
+    label: '',
+    descriptions: [],
+    onChange: () => {},
+    formatValue: () => {},
+    parseValue: () => {},
+    fieldName: '',
+    children: '',
+  };
+
+  render() {
+    const {
+      containerClasses,
+      labelClasses,
+      inputWrapperClasses,
+      descriptionClasses,
+      withErrorHint,
+      withPercent,
+      isRadio,
+      label,
+      descriptions,
+      onChange,
+      formatValue,
+      parseValue,
+      fieldName,
+      children,
+    } = this.props;
+    return (
+      <div className={cx('form-group-container', `${containerClasses}`)}>
+        <span className={cx('form-group-column', 'switch-auto-analysis-label', `${labelClasses}`)}>
+          {label}
+        </span>
+        <div className={cx('form-group-column', `${inputWrapperClasses}`)}>
+          {withPercent && <span className={cx('percent-icon')}>%</span>}
```

Comment:
I think it's better to expand input component with this functionality rather than adding something here using tricky positioning

Reply:
Fixed